ai-platform-unified/notebooks/ucaip_customjob_image.ipynb

```diff
@@ -0,0 +1,2540 @@
+{
```

Comment:
<p>These links should be update:</p><p> </p><ul><li>First should be https://colab.research.google.com/github/GoogleCloudPlatform/ai-platform-samples/master/ai-platform-unified/notebooks/ucaip_customjob_image.ipynb</li><li>Second should be https://github.com/GoogleCloudPlatform/ai-platform-samples/master/ai-platform-unified/notebooks/ucaip_customjob_image.ipynb</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>updated, but seems the links will be working only when after merged</p>

Reply:
Since you renamed the notebook, you'll need to update these links again (to use `custom_image_classification_model_for_online_prediction.ipynb` at the end of the URLs) 😄